papergrid/src/lib.rs

```diff
@@ -919,13 +952,56 @@ impl std::fmt::Display for Grid {
                 inner_border,
                 top_border,
                 bottom_border,
+                self.margin,
             )?;
         }
 
+        for _ in 0..self.margin.bottom.size {
+            writeln(f, |f| {
+                repeat_char(f, self.margin.bottom.fill, margin_row_widths)
+            })?
+        }
+
         Ok(())
     }
 }
 
+fn margin_line_widths(
```

Comment:
It could be replaced by `Grid::total_width` but it's not merged yet (#77)

Reply:
👌🏻
It's up to you, merge #77 or #82 first )

Reply:
I did merged it a few minutes ago.
You can check that function out.

But don't worry too much I'll take a look at it myself if you will have no chance to.

Reply:
Still not looked on improvements we could take to be more "adaptive" in papergrid rendering logic.

Reply:
👌🏻